<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>gestion_youssef</class>
 <widget class="QMainWindow" name="gestion_youssef">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-image: url(:/res/img/bg.png);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="studio">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>741</width>
      <height>481</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>ajouter studio</string>
     </attribute>
     <widget class="QLineEdit" name="id">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>180</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QComboBox" name="salle">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>280</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>3</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>4</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>5</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_20">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>280</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Salle</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_21">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>180</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>  ID</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_22">
      <property name="geometry">
       <rect>
        <x>190</x>
        <y>230</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Dispo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dispo">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>230</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>            --</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>oui</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>non</string>
       </property>
      </item>
     </widget>
     <widget class="QLabel" name="label_23">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>40</y>
        <width>231</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#f90000;&quot;&gt;Ajouter Studio&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="ajouter">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifer</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>gestion studio</string>
     </attribute>
     <widget class="QLineEdit" name="read_studio">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>20</y>
        <width>241</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ff1601;&quot;&gt;Gestion Studio&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="journaliste_sel_col">
      <property name="geometry">
       <rect>
        <x>1051</x>
        <y>150</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>All</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nom</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Specialite</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nb Articles</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>410</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>envoyer mail</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_studio">
      <property name="geometry">
       <rect>
        <x>542</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Par date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>ID</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Salle</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Dispo</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="afficher">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QTableView" name="Afficher_2">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>691</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ajouter equipement</string>
     </attribute>
     <widget class="QLabel" name="label_24">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>320</y>
        <width>61</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>name</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_25">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>220</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>nb</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="nb">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>220</y>
        <width>113</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_3">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>290</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Modifer</string>
      </property>
     </widget>
     <widget class="QComboBox" name="name">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>320</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>lumiere</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>micro</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>eau</string>
       </property>
      </item>
     </widget>
     <widget class="QPushButton" name="ajouter_2">
      <property name="geometry">
       <rect>
        <x>460</x>
        <y>220</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>ajouter</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_26">
      <property name="geometry">
       <rect>
        <x>210</x>
        <y>80</y>
        <width>311</width>
        <height>41</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#f90000;&quot;&gt;Ajouter Equipement&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_27">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>270</y>
        <width>47</width>
        <height>14</height>
       </rect>
      </property>
      <property name="text">
       <string>Dispo</string>
      </property>
     </widget>
     <widget class="QComboBox" name="dispo_2">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>270</y>
        <width>111</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>oui</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>non</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>gestion equipement</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>20</y>
        <width>301</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:18pt; font-weight:600; color:#ff1601;&quot;&gt;Gestion Equipement&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_equipement">
      <property name="geometry">
       <rect>
        <x>542</x>
        <y>30</y>
        <width>81</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Par date</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NB</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>NAME</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>DISPO</string>
       </property>
      </item>
     </widget>
     <widget class="QTableView" name="Afficher_4">
      <property name="geometry">
       <rect>
        <x>20</x>
        <y>70</y>
        <width>691</width>
        <height>321</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="afficher_3">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>afficher</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp_3">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="read_equipement">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>30</y>
        <width>121</width>
        <height>24</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Recherche</string>
      </property>
      <property name="clearButtonEnabled">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_10">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>410</y>
        <width>331</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>envoyer mail</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pdfEquip">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>400</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Export PDF</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actiondeconnecter"/>
  </widget>
  <action name="actiondeconnecter">
   <property name="icon">
    <iconset>
     <normaloff>img/Windows-Log-Off-icon.png</normaloff>
     <normalon>:/res/img/Windows-Log-Off-icon.png</normalon>img/Windows-Log-Off-icon.png</iconset>
   </property>
   <property name="text">
    <string>Se deconnecter</string>
   </property>
   <property name="toolTip">
    <string>deconnecter</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
